R/get_phylopic.R

```diff
@@ -2,25 +2,30 @@
 #'
 #' This retrieves a PhyloPic silhouette as a vectorized or rasterized object.
 #'
-#' @param uuid A PhyloPic image uuid.
-#' @param format Format of the image. To return a vectorized image, use
-#'   "vector". To return a rasterized image, use one of 512, 1024, or 1536.
-#'   Rasterized thumbnails can be returned by using 64, 128, or 192. Finally,
-#'   using "twitter" will return a rasterized image that includes the PhyloPic
-#'   logo and is formatted for posting on social media.
-#' @return If `format` is "vector", a [Picture][grImport2::Picture-class]
-#'   object is returned. Otherwise, a png array representing the rasterized
-#'   image is returned. Either way, the uuid and download url are included as
-#'   the "uuid" and "url" attributes, respectively.
+#' @param uuid \code{character}. A PhyloPic image uuid.
+#' @param format \code{character}. Format of the image. To return a vectorized
+#'   image, use "vector". To return a rasterized image, use "raster" and specify
+#'   a desired `height`. Rasterized square thumbnails can be returned by using
+#'   "64", "128", or "192". Finally, using "twitter" will return a rasterized
```

Comment:
Are these thumbnails always available? Would it be better just to have vector, raster and twitter here? I guess 64, 128, and 192 options would be better placed in height?

Reply:
These thumbnails are special in that they are square instead of rectangular. This matters for the background, but when we remove the background, it doesn't really matter at all. My original intent here was to just have the character options (vector, raster, and twitter), but I realized this small distinction, so I kept the thumbnail options for `format`. Thoughts?

Reply:
If thumbnails are a different image type (i.e. unique from a raster image of the same dimensions), perhaps `format` should have a "thumbnail" option and you define the height accordingly in `height`. I *think* this is a little more intuitive.